__tests__/label.test.js

```diff
@@ -16,6 +16,16 @@ describe('Label', () => {
     await clearDatabaseState(app);
   });
 
+  describe('get', () => {
```

Comment:
by resorce routing rules we have names index, create, update, show, delete. This one is index, better to stick with it